build_misc/templates/pkg.control

```diff
@@ -0,0 +1,8 @@
+Package: @pkg@
+Architecture: @DEB_ARCH@
+Maintainer: @DEB_MAINTAINER@
+Version: @DEB_@PKG@_V@
+Section: @Section@
+Priority: optional
+Description: @DEB_DESCRIPTION@
+@DEB_EXTENDED_DESCRIPTION@
```

Comment:
Add a new line here.

Reply:
@DEB_EXTENDED_DESCRIPTION@ already has a new line at end of file.